### PR TITLE
Simple `integrationtest` to verify backup&recovery are working

### PR DESCRIPTION
**Pull request by dpc**

### CODE REVIEW

**Review by jkitman — APPROVED** [review]
Looks great, glad the framework worked for your test!

File: integrationtests/tests/tests.rs [hunk 1]
```diff
@@ -970,3 +971,53 @@ async fn rejoin_consensus_threshold_peers() -> Result<()> {
...
+            .unwrap()
+            .unwrap();
+
+        assert_eq!(user_send.total_coins().await, sats(5000));
+
+        let ecash = fed.spend_ecash(&user_send, sats(3500)).await;
+        user_receive.client.reissue(ecash, rng()).await.unwrap();
+        fed.run_consensus_epochs(2).await; // process transaction + sign new coins
+
+        user_send
```
Comment (jkitman) [inline integrationtests/tests/tests.rs:1011]: How come you need to restore twice?

File: integrationtests/tests/tests.rs [hunk 2]
```diff
@@ -970,3 +971,53 @@ async fn rejoin_consensus_threshold_peers() -> Result<()> {
     })
     .await
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn ecash_can_be_recovered() -> Result<()> {
+    test(4, |fed, user_send, bitcoin, _, _| async move {
```
Comment (jkitman) [inline integrationtests/tests/tests.rs:977]: You might be able to use 2 peers which makes the tests faster.

File: integrationtests/tests/tests.rs (hunk 1 above)
Comment (dpc) [reply integrationtests/tests/tests.rs:1011]: One can always keep repating:

* make an operation,
* restore, check if correctly restored w.r.t operation from the previous step

I just did it twice here.

File: integrationtests/tests/tests.rs (hunk 2 above)
Comment (dpc) [reply integrationtests/tests/tests.rs:977]: Can you explain?

Comment (jkitman) [reply integrationtests/tests/tests.rs:977]: I mean just changing the 4 to a 2 will reduce the size of the federation.  Usually I used 2 peers unless necessary.

Comment (dpc) [reply integrationtests/tests/tests.rs:977]: Oh. Thanks! #1131

**Pull request merged**